src/main/resources/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration/help-credentialsId.html

```diff
@@ -1,8 +1,10 @@
 <div>
-    <p>Adding credentials makes it easier for users to set up Jenkins projects. Users will be able to choose from these
-        credentials to allow Jenkins to authenticate with Bitbucket and retrieve their projects. You should add
-        credentials in the form of a 'username with password'.</p>
+    <p>When running a job, Jenkins uses credentials to check out the source code for builds. You can provide Jenkins
+        with credentials here, or your users can provide them when creating a Jenkins job.</p>
 
-    <p>It's not mandatory to add credentials. If you choose not to users will be able to add their own when setting up a
-        project.</p>
+    <p>Of these options, we recommend providing credentials here. Doing this makes it easier for users to create jobs.
+        Instead of having to add their own credentials, they can choose from the ones you have added. To make job
+        creation even easier, the credentials you choose here will also be selected for them by default.</p>
+
+    <p>Credentials must be added as an username and password.</p>
```

Comment:
```suggestion
    <p>Credentials must be added as an Username with password.</p>
```
As that is what the credential type is called in Jenkins.

Reply:
I might do this with quotes to emphasize the option to choose here

Reply:
@mhenschle-atl there is a small typo in the suggestion. Please use this one instead. 

Also, is it possible to use 'bold' instead here? It's what we usually use for things like menu items we want to draw attention to. 

```suggestion
    <p>Credentials must be added as a <b>Username and password</b>.</p>
```

Reply:
Yep bold works here. I'll add it.